server/src/main/java/io/spine/server/SubscriptionService.java

```diff
@@ -120,6 +121,9 @@ private BoundedContext selectBoundedContext(Subscription subscription) {
     private BoundedContext selectBoundedContext(Target target) {
         TypeUrl type = TypeUrl.parse(target.getType());
         BoundedContext result = typeToContextMap.get(type);
+        if (result == null) {
+            throw newIllegalArgumentException("Trying to subscribe to an unknown type: %s", type);
```

Comment:
This method is going to be called from `cancel` via `selectBoundedContext(Subscription subscription)`. So the text is misleading.

Also, I don't think that's a good idea to throw an exception. I'd rather return `Optional.empty()`. And if called from `subscribe` I would throw an exception (and return a respective error via gRPC observer's `onError`). But if called from `cancel` I would emit a warning, as such a use case isn't a show-stopper.